bundles/python/python_lexer.moon

```diff
@@ -83,11 +85,49 @@ howl.aux.lpeg_lexer ->
 
   decorator = c 'preproc', P'@' * name * ('.' * name)^0
 
-  P {
+  f_prefix = c 'special', any {
+    S'rR' * S'fF'
+    S'fF' * S'rR'
+    S'fF'
+  }
+
+  format_conv = c('operator', P'!') * c('special', S'sra')
+  format_number = any {
+    c 'number', integer
+    c('operator', P'{') * ((V'all' + space + P 1) - P'}')^0 * c('operator', P'}')
+  }
+  format_part = any {
+    format_number
+    c 'special', (P 1) - S'{}'
+  }
+  format_spec = c('operator', ':') * format_part^0
+
+  string_kinds =
+    sq_string: "'"
+    dq_string: '"'
+    tsq_string: "'''"
+    tdq_string: '"""'
+  interpolations = {}
+  f_strings = {}
+  for kind, quote in pairs string_kinds
```

Comment:
This is not right - the order of keys from `pairs` is not specified (and even if it was, and tables had a set order the above definition would still not work right). The triple quotes need to be matched first, otherwise the single variants will also match with predictably bad results. It so happens on my machine that the order ends up being `'''`, `"`, `"""`, `'`. So by luck the triple example in the example file actually works. The example file does by chance not contain an example of triple double quoted strings, but as is expected they do not work.